.github/workflows/workflow.yml

```diff
@@ -79,6 +79,45 @@ jobs:
       - run: |
           opam exec -- dune build --profile=dev --only-packages http,cohttp,cohttp-lwt,cohttp-lwt-unix,cohttp-server-lwt-unix,cohttp-mirage,cohttp-async,cohttp-curl-async,cohttp-curl-lwt,cohttp-curl,cohttp-top @install
           opam exec -- dune build -j1 bench/ examples/ cohttp-curl-async/bin/ cohttp-curl-lwt/bin/
-
       - run: opam exec -- dune runtest --profile=dev cohttp cohttp-async cohttp-curl cohttp-curl-async cohttp-curl-lwt cohttp-lwt cohttp-lwt-unix cohttp-mirage cohttp-server-lwt-unix cohttp-top http
 
+  build-test-cohttp-eio:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+        ocaml-compiler:
+          - ocaml-variants.5.0.0+trunk
+        local-packages:
+          - |
+            http.opam
+            cohttp-eio.opam
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - run: sudo apt-get install ncat
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+
+      - run: brew install nmap
+        if: ${{ matrix.os == 'macos-latest' }}
```

Comment:
`ncat/nmap/nc` either should come with a `conf-` package added as a test dependency or we should find a different way to run the tests

Reply:
Will create a PR to add `conf-nmap.opam` to opam-repository.